app/views/state_file/questions/az_review/edit.html.erb

```diff
@@ -59,7 +59,7 @@
       <% if current_intake.az321_contributions.present? %>
         <% current_intake.az321_contributions.each do |contribution| %>
           <p class="text--bold spacing-below-5"><%=t(".az321_contribution_name", charity_name: contribution.charity_name, charity_code: contribution.charity_code) %></p>
-          <p><%= number_to_currency(contribution.amount) || t("general.none") %></p>
+          <p><%= number_to_currency(contribution.amount, precision: 2) || t("general.none") %></p>
```

Comment:
if `contribution.amount` is `nil`, what is returned? wondering when `t("general.none")` is shown

Reply:
it just say "None"

Reply:
oh i guess `number_to_currency(nil, precision: 2)` is returning `nil` to allow the "None" text to show up